src/utils/__tests__/lesson-metadata.test.ts

```diff
@@ -0,0 +1,135 @@
+import {LessonResource} from 'types'
+import {mergeLessonMetadata} from '../lesson-metadata'
```

Comment:
test coverage for the different ways that data is merged in `mergeLessonMetadata`